javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,71 @@
+// Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function getShortestWord() {
+    let shortest = 0;
+    for (let i = 0; i < danishWords.length; i++) {
+        if (danishWords[i].length < danishWords[shortest].length) {
+            shortest = i;
+        }
+    }
+    return danishWords[shortest];
+}
+
+shortest = getShortestWord();
+console.log(shortest);
+
+// Find and count the Danish letters
+
+const uniqueLetter = ["å", "æ", "ø"];
+
+function countLetters(str) {
+    let result = { total: 0 };
+
+    for (let i = 0; i < str.length; i++) {
+        if (uniqueLetter.includes(str[i])) {
+            result.total = result.total + 1;
+            if (result.hasOwnProperty(str[i])) {
+                result[str[i]] = result[str[i]] + 1;
```

Comment:
in here one might prefer using += to increase and update the value 

 `result[str[i]] += 1;`